packages/core/src/time/time.element.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { html } from 'lit-element';
+import { globalStyle } from '@clr/core/internal';
+import { CdsControl } from '@clr/core/forms';
+import { inputStyles } from '@clr/core/input';
+import { ClarityIcons, clockIcon } from '@clr/core/icon';
+import '@clr/core/icon';
+import { styles as globalStyles } from './time.global.css.js';
+
+ClarityIcons.addIcons(clockIcon);
+
+/**
+ * Time Input
+ *
+ * ```typescript
+ * import '@clr/core/time/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-time>
+ *   <label>Time</label>
+ *   <input type="time" />
+ *   <cds-control-message>message text</cds-control-message>
+ * </cds-time>
+ * ```
+ *
+ * @element cds-time
+ * @slot default - For projecting time input and label
+ */
+export class CdsTime extends CdsControl {
+  @globalStyle() protected globalStyles = globalStyles;
+
+  protected get suffixDefaultTemplate() {
+    return html`<cds-control-action readonly><cds-icon shape="clock"></cds-icon></cds-control-action>`;
+  }
```

Comment:
Time similar to search does not provide any extra functionality to the plain native but normalizes the styles across browsers to be consistent.